RELEASE/scripts/autoscend/auto_acquire.ash

```diff
@@ -765,7 +774,7 @@ int handlePulls(int day)
 			{
 				pullXWhenHaveY($item[Fake Washboard], 1, 0);
 			}
-			if((item_amount($item[Fake Washboard]) == 0) && (closet_amount($item[Fake Washboard]) == 0))
+			if((item_amount($item[Fake Washboard]) == 0) && (closet_amount($item[Fake Washboard]) == 0) && !auto_turbo())
```

Comment:
there's an else immediately after this that assumes you have a fake washboard, but you might not if you're not turbo

Reply:
Moved the turbo check inside the first if so it no longer affects the else.